src/App.vue

```diff
@@ -16,29 +14,31 @@
 </template>
 
 <script lang="ts">
-import { ref, defineComponent, watchEffect, watch, computed } from 'vue'
-import { useRoute } from 'vue-router';
-import { version } from './../package.json'
-import SearchBar, {SearchOption} from './components/SearchBar.vue'
-import { searchablePages } from './router/navigation';
+import { ref, defineComponent, watchEffect, watch, computed } from "vue";
```

Comment:
this is probably something we should write in contributing and add to our lint config
I am talking about using double quotes for strings

Reply:
Sounds good